src/20_Components/amp-fx-flying-carpet.html

```diff
@@ -33,13 +45,32 @@
 
      The text that follows is to ensure that the `amp-fx-flying-carpet` is placed after the first viewport.
   -->
+  <p>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis venenatis, metus a tristique aliquet, enim nibh efficitur sem, ut iaculis urna justo eu diam. Nullam cursus sapien et sodales posuere. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Suspendisse potenti. Donec vitae ornare risus. Maecenas eleifend ante vel dui laoreet, et porttitor libero rutrum. Nam arcu mi, ullamcorper at risus et, pulvinar ultrices erat. In pellentesque sem vel purus auctor, ut venenatis tellus tristique. Phasellus molestie diam orci, nec gravida turpis bibendum ut. Sed sagittis aliquet lorem sed dictum.
+  </p>
+  <!--
+    `amp-fx-flying-carpet` can be used to display ads.
+
+    Use `height` parameter to specify the height of the flying carpets "window".    
+  -->
+    <amp-fx-flying-carpet height="300px">
+      <amp-ad width="300" 
+                class="align-right"
+                height="250"
+                type="a9"
+                data-aax_size="300x250"
+                data-aax_pubname="test123"
+                data-aax_src="302">
+      </amp-ad>
+    </amp-fx-flying-carpet>
+ 
+  <!-- -->
     <p>
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis venenatis, metus a tristique aliquet, enim nibh efficitur sem, ut iaculis urna justo eu diam. Nullam cursus sapien et sodales posuere. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Suspendisse potenti. Donec vitae ornare risus. Maecenas eleifend ante vel dui laoreet, et porttitor libero rutrum. Nam arcu mi, ullamcorper at risus et, pulvinar ultrices erat. In pellentesque sem vel purus auctor, ut venenatis tellus tristique. Phasellus molestie diam orci, nec gravida turpis bibendum ut. Sed sagittis aliquet lorem sed dictum.
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis venenatis, metus a tristique aliquet, enim nibh efficitur sem, ut iaculis urna justo eu diam. Nullam cursus sapien et sodales posuere. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Suspendisse potenti. Donec vitae ornare risus. Maecenas eleifend ante vel dui laoreet, et porttitor libero rutrum. Nam arcu mi, ullamcorper at risus et, pulvinar ultrices erat. In pellentesque sem vel purus auctor, ut venenatis tellus tristique. Phasellus molestie diam orci, nec gravida turpis bibendum ut. Sed sagittis aliquet lorem sed dictum.
     </p>
 
   <!--
-    `amp-fx-flying-carpet` can be used to display images.
-    Use `height` parameter to specify the height of the flying carpets "window".
+    `amp-fx-flying-carpet` can be also used to display images.
 
     Use vw/vh to size elements inside the `amp-fx-flying-carpet` relative to the size of the viewport. This will allow the best scrolling effect having the element using the entire viewport. 
```

Comment:
suggestions:  

* mark vw and vh as code `vw`/`vh` 
* s/This will allow the best scrolling effect having the element using the entire viewport./Having the element cover the entire viewport provides a nice scrolling effect./